tests/test_importer_utils.py

```diff
@@ -79,6 +79,16 @@ def test_get_text_inside_brackets_succeed(self):
         output = "Kulla Gunnarstorp 1:21"
         self.assertEqual(utils.get_text_inside_brackets(text), output)
 
+    def test_get_text_inside_brackets_nested_one(self):
+        text = "outsidetext (foo (cat))"
+        output = "foo (cat)"
+        self.assertEqual(utils.get_text_inside_brackets(text), output)
+
+    def test_get_text_inside_brackets_nested_two(self):
+        text = "text (foo (bar (cat))) around"
+        output = "foo (bar (cat))"
+        self.assertEqual(utils.get_text_inside_brackets(text), output)
```

Comment:
It does return `"foo) mid (bar"`.

I guess in case of multiple instances it makes most sense to return a list, so I'll try going with that.